test/utils/npm-client.spec.ts

```
@@ -0,0 +1,39 @@
+import { jest } from '@jest/globals'
+
+import type { NpmClient } from 'eslint-plugin-import-x/utils'
+
+describe('npm-client', () => {
+  let getNpmClient!: () => NpmClient
+  let getNpmInstallCommand!: (packageName: string) => string
+
+  beforeEach(async () => {
+    delete process.env.npm_config_user_agent
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)